src/ndb/ndb_common.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Roman Gezikov <rgezikov@exafore.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#ifndef SRC_NDB_COMMON_H_
+#define SRC_NDB_COMMON_H_
+
+#include <libswiftnav/common.h>
+#include <libswiftnav/ephemeris.h>
```

Comment:
What do you think about moving the datatype-specific includes to the respective header files?